### ISSUE TITLE
item name = Lookup Failed!

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#378]
This is only happening to Apparel, and only a small subset (modded and normal).

---

## Details 
I have verified install file integrity. I have not systematically removed mods to see if one is causing it, but I don't think this is the case.

It's only affecting about 20 out of 100+ items

## Game Version
1.7.36

## Language
English (EN)

## New Game
No

## Reproduction Steps
I don't know how to reproduce it.

**Comment by Constellation-VASCO** [comment]
I did remove clothmod.esm, with no change in the problem.

I also completely deleted the data/strings directory, also no effect.

**Comment by Pickysaurus** [comment]
Does this only happen with the mod installed? Is it still a problem with no mods loaded? If so please provide a specific item for us to test.

**Comment by Constellation-VASCO** [comment]
I just tried running from the steam app instead of sfse_loader, and it did solve the issue. I'll go through each mod (23 ESMs, 3 ESPs) to narrow it down.

**Comment by Constellation-VASCO** [comment]
Well, that was easy. I started with commenting out everything in my plugins.txt, and the problem went away, so I then added back only the 3 ESPs and then ran through all iterations of just those. The problem is *xatmosLegendaryDrip.esp. No other combo produces the problem. I'll relay it to that mod dev and let this ticket be closed.

**Comment by Constellation-VASCO** [comment]
Correct if I'm wrong, but ElminsterAU strongly discouraged usage of ESPs for Starfield and disabled their creation in SF1Edit. So, don't use any ESPs for Starfield.

**Comment by Constellation-VASCO** [comment]
I've tried to delete everything with .ESP, I've also deleted cache, shaders, restarted my computer, deleted Starfield.ESM, verified, restarted again and yet I have a issue with the skills. worked perfectly fine until I installed a mod that changed skills, I uninstalled it after it happened the first time but the problem is still there. I've tried using Xtranslator but everyting is in english so no luck there either.
Even tried starting new game but no luck.
Even tried disabeling all mods but I'm all out of clues of what to do.
